compair/__init__.py

```diff
@@ -87,10 +87,21 @@ def create_app(conf=config, settings_override=None, skip_endpoints=False, skip_a
 
     app.logger.debug("Application Configuration: " + str(app.config))
 
+    # setup celery scheduled tasks
+    if app.config.get('DEMO_INSTALLATION', False):
+        from celery.schedules import crontab
+
+        app.config['CELERYBEAT_SCHEDULE'] = {}
+        app.config['CELERYBEAT_SCHEDULE']['reset-demo-data-daily'] = {
+            'task': "compair.tasks.demo.reset_demo",
+            'schedule': crontab(hour=0, minute=0)
```

Comment:
maybe change the default hour to 3am. quite a lot of people are still active around midnight